sw/blas/gemm/src/gemm.h

```diff
@@ -35,9 +35,9 @@ static inline double multiply_opt(double multiplicand, double multiplier) {
         return 0;
 }
 
-void gemm_fp32_baseline(uint32_t M, uint32_t N, uint32_t K, float* A,
-                        uint32_t ldA, uint32_t ta, float* B, uint32_t ldB,
-                        uint32_t tb, float* C, uint32_t ldC, float BETA) {
+void gemm_fp32_naive(uint32_t M, uint32_t N, uint32_t K, float* A, uint32_t ldA,
```

Comment:
What is the difference between "naive" and "baseline" variants?

Reply:
The idea is that the baseline kernels should be a "fair" baseline against the SSR/frep extensions i.e. the best you can get out of Snitch without using the ISA extensions.